pyvocz/views.py

```diff
@@ -135,31 +135,54 @@ def calendar(year=None):
                            first_year=first_year, last_year=last_year)
 
 
-@route('/<series_slug>/')
-def series(series_slug):
+@route('/<series_slug>/', defaults={'year': None})
+@route('/<series_slug>/<int:year>/')
```

Comment:
Oh, I absolutely forgot about that when I was finishing it on the weekend. Will do!